discord_bot.py

```diff
@@ -68,14 +70,7 @@ async def on_message(message):
                 convo_id = get_conversation_id(conn, original_message_id) if original_message_id else None
                 logging.info(f"Received conversation id from DB: {convo_id}")
 
-            vectara = VectaraQuery(
-                customer_id=os.getenv("VECTARA_CUSTOMER_ID"),
-                corpus_ids=os.getenv("VECTARA_CORPUS_IDS").split(','),
-                api_key=os.getenv("VECTARA_API_KEY"),
-                prompt_name=vectara_prompt,
-                conv_id=convo_id
-            )
-            vectara_convo_id, response = vectara.submit_query(message_content)
+            vectara_convo_id, response = query_vectara(message_content, convo_id, vectara_prompt, bot_type="discord")
```

Comment:
Nit: why are we calling this variable "convo_id" instead of "conv_id" - maybe update to make it consistent across the code?